sdk/schemaregistry/azure-schemaregistry/samples/sync_samples/schema_registry.py

```diff
@@ -45,7 +46,27 @@
 SCHEMA_GROUP = os.environ['SCHEMA_REGISTRY_GROUP']
 SCHEMA_NAME = 'your-schema-name'
 SERIALIZATION_TYPE = SerializationType.AVRO
-SCHEMA_STRING = """{"namespace":"example.avro","type":"record","name":"User","fields":[{"name":"name","type":"string"},{"name":"favorite_number","type":["int","null"]},{"name":"favorite_color","type":["string","null"]}]}"""
+
+SCHEMA_JSON = {
+    "namespace": "example.avro",
+    "type": "record",
+    "name": "User",
+    "fields": [
+        {
+            "name": "name",
+            "type": "string"
+        },
+        {
+            "name": "favorite_number",
+            "type": ["int", "null"]},
+        {
+            "name": "favorite_color",
+            "type": ["string", "null"]
+        }
+    ]
+}
+
+SCHEMA_STRING = json.dumps(SCHEMA_JSON, separators=(',', ':'))
```

Comment:
the service is using compact encoding which doesn't have space in the json string.